docker/Dockerfile

```diff
@@ -1,3 +1,35 @@
-FROM ghcr.io/navikt/baseimages/temurin:17
+FROM navikt/java:common AS java-common
+
+FROM eclipse-temurin:17-jre
```

Comment:
Har lyst til å bytte ut denne med ein temurin-alpine-variant, mistenkjer at dette baseimaget inneheld litt for mykje.